noggin/templates/spamcheck.html

```diff
@@ -0,0 +1,58 @@
+{% extends "main.html" %}
+
+{% block content %}
+  {{ super() }}
+
+  <div class="bg-light border-bottom py-5">
+    <div class="container section">
+      <div class="row">
+        <div class="col-md-7">
+          {{ front_page_blurb() }}
+        </div>
+        <div class="col">
+          <div class="alert alert-{% if g.current_user.status_note == 'spamcheck_denied' %}warning{% else %}info{% endif %}" role="alert">
+            {% if g.current_user.status_note == "spamcheck_awaiting" %}
+              <h4 class="alert-heading">
+                {{ _("Your account is being checked") }}
```

Comment:
Again, why does it have to be wrapped in a Jinja expression like this? I'm used to messages being extracted for translation from templates and replaced with localized versions when rendered, just as they are.

Reply:
To clarify my question, why is this...:

````
{{ _("Your account is being checked") }}
````

...not this?

````
Your account is being checked
````

Reply:
Not sure what you mean, do you have an example? I'm following [this doc](https://jinja.palletsprojects.com/en/2.11.x/extensions/#new-style-gettext).

Reply:
@nphilipp for translations :)

Reply:
I've created a PR covering "we should do l10n in templates differently" here: https://github.com/fedora-infra/noggin/issues/317